### ISSUE TITLE
[BUG]: with cropStart and cropEnd is set, audio plays from the original start position (0ms) when repeat is set to true

### ISSUE THREAD

**Issue by baky30** [TheWidlarzGroup/react-native-video#3518]
### Version

v6 (Beta)

### What platforms are you having the problem on?

iOS

### Architecture

Old architecture

### What happened?

Setting repeat to true with cropStart and cropEnd values, audio plays from the start ignoring cropStart
It works when audio is being played for the first time, but when it reaches cropEnd value audio starts from the 0ms value.
Meanwhile cropEnd works: Audio stops playing when it reaches cropEnd and restarts from the top (0ms)
```
<Video
          source={{
            uri: selectedAudio.path,
            cropStart: 2000,
            cropEnd: 4000,
          }}
          repeat={true}
          ...
 />     
```
          

### Reproduction

Step to reproduce this bug are: 

1. Select 5 second audio file
2. Set cropStart to 2000 and cropEnd to 4000
3. Enable repeat (repeat={true})

**Comment by KrzysztofMoch** [comment]
Hey @baky30 I made PR to fix it, you can test it and check if it will be working for you 🙌

**Comment by KrzysztofMoch** [comment]
This will included in `6.0.0-beta.6`
